queries.yaml

```diff
@@ -1052,7 +1052,7 @@ post_process:
       end_zoom: 9
       items_matching: { kind: locality }
       max_items: 1
-      grid_width: 3
```

Comment:
This one feels a little small in the build, so we're investigating increasing it to "4" or changing "max_items" to 2.

Reply:
After we tried some variations locally, in d602ff1 I switched the first zoom range to grid_width "6" (x 4 for the metatile) with max_items 1.

Note that this makes the first and second zoom ranges in here do the same thing. If this ends up looking OK, we can consolidate the two ranges into one wider range.